### PR TITLE
Error al personalizar un worskpace

### PR DESCRIPTION
**Pull request by fproldan**
https://github.com/fproldan/DiamoERP/issues/254

### CODE REVIEW

File: erpnext/hr/workspace/hr/hr.json [hunk 1]
```diff
@@ -881,12 +881,5 @@
    "link_to": "Monthly Attendance Sheet",
    "type": "Report"
   },
```
Comment (fproldan) [inline erpnext/hr/workspace/hr/hr.json]: @ValentinaPruvost 
https://github.com/fproldan/DiamoERP/issues/254#issuecomment-1070985401

image

**Review by lucasvazq — APPROVED** [review]
hay conflictos

**Comment by ValentinaPruvost** [comment]
@fproldan acá fallaron unos backports

**Comment by fproldan** [comment]
@ValentinaPruvost agregados

**Pull request merged**